readme.md

```diff
@@ -541,7 +541,330 @@ namespace Azure.Service.Models
 
 ### Customize serialization/deserialization methods
 
-Use the [Replace any generated member](#replace-any-generated-member) approach to replace Serialize/Deserialize method with a custom implementation.
+If you want to change the property name that serializes into the JSON or deserializes from the JSON, you could define your own property name using the `CodeGenMemberSerialization` attribute.
+
+<details>
+
+Define a partial class and the property you want to change in the partial class, and add `CodeGenMemberSerialization` attribute to it:
```

Comment:
This is a very simple use case therefore it could apply to this pattern.
In the use case now, if we just put all the "generated code before" here, the document will be too long to read.
I could add some more explanation by describing a scenario and how we could do it and how the code CHANGES